packages/core/src/button/button.element.ts

```diff
@@ -139,19 +139,12 @@ export class CdsButton extends CdsBaseButton {
           ? html`${iconSpinnerError}`
           : ''}${loadingState === ClrLoadingState.LOADING ? html`${iconSpinner}` : ''}${loadingState ===
         ClrLoadingState.DEFAULT
-          ? html`<slot @slotchange=${() => this.wrapTextNodes()}></slot>`
+          ? html`<slot @slotchange=${() => spanWrapper(this.childNodes)}></slot>`
           : ''}${this.hiddenButtonTemplate}
       </div>
     </div>`;
   }
 
-  private wrapTextNodes() {
```

Comment:
Did we have any tests or do we need any tests with this? What does test coverage say?

Reply:
There weren’t tests for the wrapTextNodes fn. It was originally just a private wrapper that handled hasFlexGapSupport check to decide if it should run the spanWrapper or not. Now we are running the spanWrapper for all browsers. both spanWrapper and hasFlexGapSupport have their own tests.